library/src/main/java/org/apache/apex/malhar/lib/wal/FSWindowDataManager.java

```diff
@@ -149,6 +149,8 @@
 
   private transient FileContext fileContext;
 
+  private SerializationBuffer serializationBuffer = new SerializationBuffer(new WindowedBlockStream());
```

Comment:
Good point @chandnisingh . 
@brightchen Also, since this is not windowed, should this be using BlockStream.